api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -242,4 +248,23 @@ private void logAdminAction(Long targetUserId, Long targetWorkspaceId, String ta
   public boolean getContactEmailTaken(String contactEmail) {
     return (!userDao.findUserByContactEmail(contactEmail).isEmpty());
   }
+
+  /**
+   * Find users matching the user's name or email
+   */
+  public List<User> findUsersBySearchString(String term, Sort sort) {
+    List<Short> dataAccessLevels;
+    if (configProvider.get().firecloud.enforceRegistered) {
+      dataAccessLevels = Collections
+          .singletonList(StorageEnums.dataAccessLevelToStorage(DataAccessLevel.REGISTERED));
```

Comment:
CONTROLLED also should be accepted, basically >= registered

Reply:
The four levels we have now are: 
```
  UNREGISTERED("unregistered"),
  
  REGISTERED("registered"),
  
  PROTECTED("protected"),
  
  REVOKED("revoked");
```
I think you mean protected?

Reply:
Yes, sorry

Reply:
Done.